wp-content/themes/core/components/pagination/loop/Controller.php

```diff
@@ -127,9 +128,9 @@ public function build_links(): array {
 			}
 
 			return [
-				'classes' => $classes,
-				'url'     => $number['url'],
-				'content' => $number['label'],
+				Link_Controller::CLASSES => $classes,
+				Link_Controller::URL     => $number['url'],
+				Link_Controller::CONTENT => $number['label'],
```

Comment:
A nice side effect of using the constants to reference our component args: it makes it clear that we're building an array of component args, even when we're not explicitly calling the component in the same place.